domains/itravel-guide.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "lylyhecker",
+    "email": "lgrant18@asu.edu"
+  },
+  "records": {
+    "CNAME": "https://lylyhecker.github.io/iTravel-Guide"
```

CNAME record is invalid as well.
```suggestion
    "CNAME": "lylyhecker.github.io"
```

---

Thank you so much